src/style.css

```diff
@@ -0,0 +1,111 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir1;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
```

Comment:
you can use the same font-family names, but different font weights for these. And it is good practice to use either numeral or text values for wont-weight, but not both of them